spec/services/openlibrary_service_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+describe OpenlibraryService do
+  let(:service) { OpenlibraryService.new }
+  let(:uri) { "#{ENV['OPEN_LIBRARY_URI']}/books" }
+
+  describe '#search_by_isbn' do
+    subject { service.search_by_isbn(isbn) }
```

Comment:
Name subject
```suggestion
    subject(:service) { service.search_by_isbn(isbn) }
```

Reply:
`search_by_isbn` returns a book, not a service :thinking: